docker/package/model.py

```diff
@@ -537,19 +538,12 @@ def fetch_sources(self, out_dir):
         shutil.copy(f"{os.path.dirname(__file__)}/tezos_setup_wizard.py", out_dir)
 
     def gen_control_file(self, deps, ubuntu_version, out):
-        run_deps = ", ".join(
-            ["acl", "tezos-client", "tezos-node"]
-            + sum(
-                [
-                    [
-                        f"tezos-{daemon}-{proto.lower()}"
-                        for daemon in ["baker", "endorser"]
-                    ]
-                    for proto in self.target_protos
-                ],
-                [],
-            )
-        )
+        run_deps_list = ["acl", "tezos-client", "tezos-node"]
+        for proto in self.target_protos:
+            run_deps_list.append(f"tezos-baker-{proto.lower()}")
+            if proto not in self.noendorser_protos:
+                run_deps_list.append(f"tezos-baker-{proto.lower()}")
```

Comment:
```suggestion
                run_deps_list.append(f"tezos-endorser-{proto.lower()}")
```

Reply:
:facepalm:

Reply:
crap, changes ended up unsaved and didn't make it to the commit :joy: